packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -162,10 +163,23 @@ export const LivenessCameraModule = (
     (!isMobileScreen || isFaceMovementChallenge);
 
   React.useEffect(() => {
-    if (canvasRef?.current && videoRef?.current && videoStream && isStartView) {
+    if (
+      canvasRef?.current &&
+      videoRef?.current &&
+      videoStream &&
+      isStartView &&
+      isMetadataLoaded
+    ) {
       drawStaticOval(canvasRef.current, videoRef.current, videoStream);
     }
-  }, [canvasRef, videoRef, videoStream, colorMode, isStartView]);
+  }, [
+    canvasRef,
+    videoRef,
+    videoStream,
+    colorMode,
+    isStartView,
+    isMetadataLoaded,
```

Comment:
The fact the dependency array here and the dependency array on line 212 are basically the same leads me to believe that we should be able to combine the behavior of these effects in a single `useEffect` which can be abstracted in to a util hook and unit tested in isolation

Reply:
Combined into a single useEffect and added a unit test to ensure the oval is drawn when metadata loads